README.md

```diff
@@ -29,11 +35,27 @@ From July 2023 `insightsengineering` packages are available on [r-universe](http
 # stable versions
 install.packages('teal.logger', repos = c('https://insightsengineering.r-universe.dev', 'https://cloud.r-project.org'))
 
+# install.packages("pak")
+pak::pak("insightsengineering/teal@*release")
+```
+
+Alternatively, you might also use the development version.
+
+```r
 # beta versions
 install.packages('teal.logger', repos = c('https://pharmaverse.r-universe.dev', 'https://cloud.r-project.org'))
+
+# install.packages("pak")
+pak::pak("insightsengineering/teal")
 ```
 
-See package vignettes `browseVignettes(package = "teal.logger")` for usage of this package.
+## Usage
```

Comment:
Can we add GIF or a simple code snapshot to showcase the package?

Reply:
Okay, maybe GIF is an overkill.
I was aiming for a quick snapshot of the package usage in the README to entice user to explore more.
The bringing over the code from vignette's `Example of Usage` is a good idea and I was thinking we can show some output as well.

Maybe something like this?
```r
library(teal.logger)
register_logger(namespace = "namespace1", level = "INFO")
logger::log_info("Hello from namespace1", namespace = "namespace1")
logger::log_warn("Hello from namespace1", namespace = "namespace1")
# [INFO] 2023-08-30 12:22:50.5723 pid:42024 token:[] namespace1 Hello from namespace1
# [WARN] 2023-08-30 12:22:50.5737 pid:42024 token:[] namespace1 Hello from namespace1

register_logger(namespace = "namespace2", level = "WARN")
logger::log_info("Hello from namespace2", namespace = "namespace2")
logger::log_warn("Hello from namespace2", namespace = "namespace2")
# [WARN] 2023-08-30 12:22:53.9459 pid:42024 token:[] namespace2 Hello from namespace2
```

Or perhaps some image to spice things up?
![image](https://github.com/insightsengineering/teal.logger/assets/8597300/f908bfe8-9011-4ee7-ae73-66afa651aa40)

What do you think?

Reply:
@donyunardi I added showcase code, as you proposed in this commit cc7da181ba3abc9f0e41768609a6c7dcfbd557a2
I also extended the use-case by log_error and log_success to show broader functionality. I think the code is enough, and presents the same as image, with the advantage of a possibility to copy the code from text, instead of the image where you actually can't copy the code

Reply:
Thanks!